modules/javafx.graphics/src/main/java/javafx/css/StyleableObjectProperty.java

```diff
@@ -137,37 +270,297 @@ public void set(T v) {
     public StyleOrigin getStyleOrigin() { return origin; }
 
     private StyleOrigin origin = null;
-    private TransitionMediatorImpl mediator = null;
+    private TransitionController<T> controller = null;
```

Comment:
I agree. Not sure if I should remove it from all of the StyleableProperty implementations, as the `origin = null` is already there.

Reply:
I know what @kevinrushforth would say: "it is better to avoid introducing unrelated changes"

as long as the new code omits `= null` we should be fine.

(rationale for omitting: explicit initialization creates an unnecessary step in the debugger)

Reply:
> I know what @kevinrushforth would say: "it is better to avoid introducing unrelated changes"

Yes, that is what I would say. :)

> as long as the new code omits `= null` we should be fine.
> 
> (rationale for omitting: explicit initialization creates an unnecessary step in the debugger)

In this case, omitting it is fine. There are cases where I prefer to explicitly initialize fields to `null`, `false`, or `0`, for clarity, even though those values are the default. I don't care that it creates an unnecessary step in your debugger.